edge-modules/TestAnalyzer/TestOperationResult.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestAnalyzer
```

Comment:
Where do we foresee TestOperationResult living for good? If other tests will use it, it should eventually be under test/ subfolder. Maybe in the Microsoft.Azure.Devices.Edge.Test.Common project?

Reply:
we will move it when start working on TestResultCoordinator.